# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mdmh/head_-n.patch
# Copyright (C) 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2005-12-16 14:43:56.000000000 -0300
+++ ./Makefile	2005-12-16 14:44:18.000000000 -0300
@@ -11,8 +11,8 @@
 mdbiff: \
 warn-auto.sh mdbiff.sh conf-home conf-play
 	cat warn-auto.sh mdbiff.sh \
-	| sed s}HOME}"`head -1 conf-home`"}g \
-	| sed s}PLAY}"`head -1 conf-play`"}g \
+	| sed s}HOME}"`head -n 1 conf-home`"}g \
+	| sed s}PLAY}"`head -n 1 conf-play`"}g \
 	> mdbiff
 	chmod 755 mdbiff
 
@@ -20,7 +20,7 @@
 warn-auto.sh conf-ld
 	( cat warn-auto.sh; \
 	echo 'main="$$1"; shift'; \
-	echo exec "`head -1 conf-ld`" \
+	echo exec "`head -n 1 conf-ld`" \
 	'-o "$$main" "$$main".o $${1+"$$@"}' \
 	) > load
 	chmod 755 load
@@ -34,7 +34,7 @@
 compile: \
 warn-auto.sh conf-cc
 	( cat warn-auto.sh; \
-	echo exec "`head -1 conf-cc`" '-c $${1+"$$@"}' \
+	echo exec "`head -n 1 conf-cc`" '-c $${1+"$$@"}' \
 	) > compile
 	chmod 755 compile
 
@@ -57,7 +57,7 @@
 
 auto_home.c: \
 auto-str conf-home
-	./auto-str auto_home `head -1 conf-home` > auto_home.c
+	./auto-str auto_home `head -n 1 conf-home` > auto_home.c
 
 auto-str: \
 load auto-str.o buffer.a error.a str.a
@@ -91,8 +91,8 @@
 systype: \
 find-systype.sh conf-cc conf-ld trycpp.c x86cpuid.c
 	( cat warn-auto.sh; \
-	echo CC=\'`head -1 conf-cc`\'; \
-	echo LD=\'`head -1 conf-ld`\'; \
+	echo CC=\'`head -n 1 conf-cc`\'; \
+	echo LD=\'`head -n 1 conf-ld`\'; \
 	cat find-systype.sh; \
 	) | sh > systype
 
